Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/MapViewModel.cs

```diff
@@ -8,31 +8,35 @@
 using Xamarin.Forms.Maps;
 using FindMyBLEDevice.Models;
 using FindMyBLEDevice.Services;
-using System.Threading.Tasks;
 using Xamarin.Forms;
-using FindMyBLEDevice.Services.Geolocation;
 using FindMyBLEDevice.Services.Database;
+using FindMyBLEDevice.Services.Geolocation;
+using System.Threading.Tasks;
+using System;
 
 namespace FindMyBLEDevice.ViewModels
 {
     public class MapViewModel : BaseViewModel
     {
+        public bool DeviceNotNull => Device != null;
+
         private readonly Xamarin.Forms.Maps.Map map;
         private readonly IGeolocation geolocation;
+        private readonly INavigator navigator;
         private readonly IDevicesStore devicesStore;
+        private bool userDeclinedPageSwitch, showingDialogue;
```

Comment:
`userDeclinedPageSwitch` is not necessary anymore, is it?

Reply:
nice catch, the userdeclined... can be removed, I simply forgot that.